src/components/recipeform.tsx

```diff
@@ -97,6 +105,9 @@ function RecipeForm(props: {
                   />
                 </Col>
               </Form.Group>
+              <button type="button" onClick={() => ingredientsRemove(index)}>
```

Comment:
Does the ingredient removal have any issues? I remember being worried about users removing an ingredient and then adding a new one.

Reply:
only going to show the add button if the nutrition is empty. deleting the delete button. there will only be one nutritionalLabel info per recipe